assets/instagram/src/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import "./App.css";
 import instagram from "./InstagramData";
-import PostsContainer from "./components/PostsContainer/PostsContainer";
```

Comment:
the app is not rendering,  it looks like you don't have the correct path.  Make sure you have your server running and are checking these types of errors.